### PR TITLE
Add to migration doc steps regards changes over metrics to the scaffolded files.

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
- Add info over changes in the default scaffold in the main.go file to address bug fixes and improvements. 

NOTE: It can just be merged after we merge: https://github.com/operator-framework/operator-sdk/pull/2606, https://github.com/operator-framework/operator-sdk/pull/2603, https://github.com/operator-framework/operator-sdk/pull/2601

### CODE REVIEW

File: doc/migration/version-upgrade-guide.md [hunk 1]
```diff
@@ -812,6 +812,104 @@ See the [CHANGELOG](https://github.com/operator-framework/operator-sdk/blob/mast
 
 ## Unreleased
 
+### Bug Fixes and Improvements for Metrics
+
+Some changes were performed in the default implementation regards the default metrics export.  In this way, update the  `cmd/main.go` file as follows.
```
Comment (jmrodri) [inline doc/migration/version-upgrade-guide.md]: rephrase. "There were some changes to the default implementation of the metrics export. These changes require the `cmd/main.go` be updated as follows."

File: doc/migration/version-upgrade-guide.md [hunk 2]
```diff
@@ -812,6 +812,104 @@ See the [CHANGELOG](https://github.com/operator-framework/operator-sdk/blob/mast
...
+	// Generate and serve custom resource specific metrics.
+	err = kubemetrics.GenerateAndServeCRMetrics(cfg, ns, filteredGVK, metricsHost, operatorMetricsPort)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+```
+
+**NOTE**: For further and detailed information check the PRs which are responsible for the above changes [#2606](https://github.com/operator-framework/operator-sdk/pull/2606),[#2603](https://github.com/operator-framework/operator-sdk/pull/2603) and [#2601](https://github.com/operator-framework/operator-sdk/pull/2601).
```
Comment (jmrodri) [inline doc/migration/version-upgrade-guide.md]: rephrase a bit, "For more information check the PRs ...."

Comment (camilamacedo86) [inline doc/migration/version-upgrade-guide.md]:
```suggestion
**NOTE**: For more information check the PRs which are responsible for the above changes [#2606](https://github.com/operator-framework/operator-sdk/pull/2606),[#2603](https://github.com/operator-framework/operator-sdk/pull/2603) and [#2601](https://github.com/operator-framework/operator-sdk/pull/2601).
```

File: doc/migration/version-upgrade-guide.md (hunk 1 above)
Comment (camilamacedo86) [inline doc/migration/version-upgrade-guide.md]:
```suggestion
There were some changes to the default implementation of the metrics export. These changes require the `cmd/main.go` be updated as follows.
```

**Pull request merged**